### ISSUE TITLE
(Beta 6) Hotbar [0-9] shortcuts reassigned when entering values in AA menu/dialogs

### ISSUE THREAD

**Issue by ubiquish** [mircearoata/SatisfactoryAreaActions#60]
Two issues, fairly easy to reproduce, likely the same root cause:

1. When naming a blueprint with a digit in the name, various AA items will be assigned (overwrite) the ones in the current hotbar.   (I assume this is simply one of those issues where the "assign functions to the hotbar" feature needs to be turned off when users are entering text in dialog boxes.)

2. Same issue when entering coordinates, fill values, or other numeric text in the other dialog boxes.   The action that is assigned to the hotbar *seemed* random to me at the time, but now that I think about it, it might have been caused by the location of my mouse cursor when inputting the data).   

_TLDR:  Hotbar assignments still happen even when accepting user input that could contain numerics, causing unintentional effects._
